### PR TITLE
bug fix Fin Control report angle

### PR DESCRIPTION
**Pull request by patapufate**
# Description
The fin_control node reports wrong values -> message published =  [0, -1, 1, 0]
and the msg have to be -> message published =  [0, -0.7, 0.7, 0]
This bug was introduced in #54 

Fixes 
This PR fixes:
- restore radian to angles conversion removed in #54

## Type of change

Please mark options that are relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How To Test?
- Launch the fin control node.
- Put the fins angles to surface
- Subscribe to /fin_control/report_angle topic

Output:
The msg have to be -> message published =  [0, -0.7, 0.7, 0]

- [x] Your changes have to be tested on hardware
- [ ] You have added unit tests

# Checklist:
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] The code builds clean without any errors or warnings
- [x] Linter tests are passing

### CODE REVIEW

File: fin_control/src/fin_control.cpp [hunk 1]
```diff
@@ -247,10 +261,9 @@ void FinControl::handleSetAngles(const fin_control::SetAngles::ConstPtr &msg)
     ROS_ERROR("Could not init bulk write %s", log);
     return;
   }
-
```
Comment (hidmic) [inline fin_control/src/fin_control.cpp:250]: @patapufate nit: unnecessary?

**Review by hidmic — COMMENTED** [review]
@patapufate line 347 also needs an update. Otherwise LGTM!

File: fin_control/src/fin_control.cpp (hunk 1 above)
Comment (patapufate) [reply fin_control/src/fin_control.cpp:250]: Done

File: fin_control/src/fin_control.cpp [hunk 2]
```diff
@@ -247,10 +259,10 @@ void FinControl::handleSetAngles(const fin_control::SetAngles::ConstPtr &msg)
...
+
```
[the remark this answers is not in the document]
Comment (patapufate) [reply fin_control/src/fin_control.cpp:262]: roslint doesn't show error. I removed it

File: fin_control/src/fin_control.cpp [hunk 3]
```diff
@@ -224,19 +223,32 @@ void FinControl::reportAngles()
...
-  )
+  float angle1_in_degrees;
+  float angle2_in_degrees;
+  float angle3_in_degrees;
+  float angle4_in_degrees;
+
+  angle1_in_degrees = radiansToDegrees(msg->f1_angle_in_radians);
+  angle2_in_degrees = radiansToDegrees(msg->f2_angle_in_radians);
+  angle3_in_degrees = radiansToDegrees(msg->f3_angle_in_radians);
+  angle4_in_degrees = radiansToDegrees(msg->f4_angle_in_radians);
```
[the remark this answers is not in the document]
Comment (patapufate) [reply fin_control/src/fin_control.cpp:244]: fixed

File: fin_control/src/fin_control.cpp [hunk 4]
```diff
@@ -198,19 +197,19 @@ void FinControl::reportAngles()
   float radianAngleFromMyWorkBench = 0.;
 
   message.header.stamp = ros::Time::now();
-  
+
   if (!reportAnglesEnabled) return;  // do not interfere with fins updating
 
-  for (int id = 0; id < numOfIDs; ++id)
+  for (int id = 1; id == numOfIDs; ++id)
```
Comment (hidmic) [inline fin_control/src/fin_control.cpp:203]: @patapufate 
```suggestion
  for (int id = 1; id <= numOfIDs; ++id)
```
!

Comment (patapufate) [reply fin_control/src/fin_control.cpp:203]: Good Catch.

**Review by hidmic — APPROVED** [review]
LGTM assuming it was tested on hardware

**Pull request merged**